app/src/common/utils/getRolesMap.js

```diff
@@ -0,0 +1,12 @@
+import { PROJECT_ROLES } from 'common/constants/projectRoles';
+
+export function getRolesMap() {
```

Comment:
Do we really need that function placed in utils? It's common method? Are there some other places where we can use it?